parse_node/parse_class_declaration.ts

```diff
@@ -167,3 +218,156 @@ func nodes_set(_v):
   pass
 `,
 }
+
+export const testDontRequireDefaultExportingAutoloads: Test = {
+  ts: `
+@autoload
+@inner
+class Foo {
+  x = 1
+}`,
+  expected: {
+    error: "Only main class can be autoloaded",
+    type: "error",
+  },
+}
+
+export const testMultipleClassesWithoutMarking: Test = {
+  ts: `
+class Foo {
+  x = 1
+}
+
+class Bar {
+  x = 1
+}`,
+  expected: {
```

Comment:
Additional note: I've made a small change in `test.ts` because tests that were expecting strings and threw an error were passing false positively when generated text matched the expected one. When an error occurs test should always fail even if contents match :)